error occurred while installing json (1.8.3)

Hi

I'm getting the following error. Unfortunately, I don't have the knowledge to debug this. Let me know if there is any more digging I can do.

```
C:\web\Ghost-Vagrant>vagrant up
Installing the 'vagrant-librarian-puppet' plugin. This can take a few minutes...
Bundler, the underlying system Vagrant uses to install plugins,
reported an error. The error is shown below. These errors are usually
caused by misconfigured plugin installations or transient network
issues. The error from Bundler is:

An error occurred while installing json (1.8.3), and Bundler cannot continue.
Make sure that `gem install json -v '1.8.3'` succeeds before bundling.

Warning: this Gemfile contains multiple primary sources. Using `source` more than once without a block is a security risk, and may result in installing unexpected gems. To resolve this warning, use a block to indicate which gems should come from the secondary source. To upgrade this warning to an error, run `bundle config disable_multisource true`.Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

    "C:/Program Files (x86)/HashiCorp/Vagrant/embedded/bin/ruby.exe" -r ./siteconf20160221-9472-1lb3fg6.rb extconf.rb
creating Makefile

make "DESTDIR=" clean

make "DESTDIR="
generating generator-i386-mingw32.def
/bin/sh: -c: line 0: syntax error near unexpected token `x86'
/bin/sh: -c: line 0: `/C/Program\ Files\ (x86)/HashiCorp/Vagrant/embedded/bin/ruby -e "puts 'EXPORTS', 'Init_generator'"  > generator-i386-mingw32.def'
make: *** [generator-i386-mingw32.def] Error 2

make failed, exit code 2

Gem files will remain installed in C:/Users/Daniel/.vagrant.d/gems/gems/json-1.8.3 for inspection.
Results logged to C:/Users/Daniel/.vagrant.d/gems/extensions/x86-mingw32/2.2.0/json-1.8.3/gem_make.out

```

Thanks

--- a/.gitignore
+++ b/.gitignore
@@ -1,12 +1,20 @@
-.vagrant
+# OS Files
 .DS_Store
+
+# Local Vagrant Files
+.vagrant
 vm_config.yml
 
-.idea/*
-*.iml
-projectFilesBackup
-
-.tmp
+# Local Librarian Files
 .librarian
 modules/*
 !modules/.gitkeep
+.tmp
+
+# Local Ghost Repo Clone
+Ghost/*
+
+# Other
+.idea/*
+*.iml
+projectFilesBackup

--- a/README.md
+++ b/README.md
@@ -97,6 +97,16 @@ If `vm_config.yml` is not present when you first run `vagrant up`, it will be au
 
 See the [working on Ghost core](https://github.com/TryGhost/Ghost/blob/master/CONTRIBUTING.md#core) section of the Ghost [contributing guide](https://github.com/TryGhost/Ghost/blob/master/CONTRIBUTING.md).
 
+## Troubleshooting
+
+If `grunt init` failes due to an `npm` error `EEXIST` regarding a `.lock` file, try:
+
+```
+rm -rf core/client/node_modules
+npm cache clear
+grunt init
+```
+
 ## Copyright & License
 
 Copyright (c) 2013-2016 Ghost Foundation - Released under the [MIT license](LICENSE).

--- a/README.md
+++ b/README.md
@@ -10,24 +10,26 @@ You will need the following applications to setup the Ghost development environm
 - [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
 - [Git](https://git-scm.com/downloads)
 
-Linux users will also need `nfs-common` and `nfs-kernel-server`
+Linux users will also need `nfs-common` and `nfs-kernel-server`:
 
 ```bash
 sudo apt-get install nfs-common nfs-kernel-sever
 ```
 
+Windows users should install Vagrant to a directory with no spaces in the path.
+
 ## Setup
 
-To get started with the Ghost development environment, you will first need to clone this repo and navigate into it:
+To get started with the Ghost development environment, you will first need to clone this repo:
 
 ```bash
 git clone git://github.com/TryGhost/Ghost-Vagrant.git
-cd Ghost-Vagrant
 ```
 
-You will now need a copy of Ghost itself:
+Clone the Ghost repo into your new local Ghost-Vagrant repo:
 
 ```bash
+cd Ghost-Vagrant
 git clone git://github.com/Tryghost/Ghost.git
 ```
 
@@ -75,7 +77,7 @@ This is useful when you want to debug what Ghost is doing when it boots up, what
 
 ## Configuring the VM
 
-You can configure various properties of the VM by creating a file named `vm_config.yml` in the root directory. The following properties are configurable:
+You can configure various properties of the VM by creating a file named `vm_config.yml` in the `Ghost-Vagrant` root directory. The following properties are configurable:
 
 - **hostname** - URI that will be used to access Ghost from the browser
 - **ip** - IP address assigned to the virtual machine

